### DISCUSSION TITLE
Flutter web pages

### DISCUSSION THREAD

**Discussion by pandoras-toolbox** [sukgu/shadow-automation-selenium#63] [Q&A]
Can "shadow-automation-selenium" be used in order to automate tests for Flutter web pages? I do **not** mean the mobile or desktop apps with Flutter.

I read this, so I thought it would work: https://stackoverflow.com/a/69369759

Here is an example Flutter web page: https://gallery.flutter.dev/#/

The elements cannot be located with the Chrome development tools and thus also not with Selenium and Java.

I tried to use "shadow-automation-selenium", started the app with web-renderer "html", it did locate more web elements than the normal Selenium finder, but I nearly could not perform a click on anything, it did not work for automating any test scenario. This means it does not work or I made something wrong.

**Comment by sukgu** [comment]
I have checked the web page. https://wonderous.app/web/#/home. Yes, you can automate it with selenium or "shadow-automation-selenium" tool.
